docs/data/joy/getting-started/usage/usage.md

```diff
@@ -1,42 +1,27 @@
 # Usage
 
-<p class="description">Learn the basics of using Joy UI components in two quick steps.</p>
+<p class="description">Learn the basics of using Joy UI components.</p>
 
-## Getting started
+## Quickstart
 
-### Set up the `CssVarsProvider` component
-
-Go to your `App.js` file and replace it with the code snippet below.
-
-You should see the text `Hello from Joy` being rendered on your browser.
+The following code snippet demonstrates a simple app that uses the Joy UI [Button](/joy-ui/react-button/) component:
 
 ```jsx
+import * as React from 'react';
 import { CssVarsProvider } from '@mui/joy/styles';
+import Button from '@mui/joy/Button';
 
-function App() {
-  return <CssVarsProvider>Hello from Joy</CssVarsProvider>;
-}
-
-export default App;
-```
-
-### Render your first component
-
-To render any Joy UI component, make sure you place them inside the `CssVarProvider` element.
-
-```diff
-import { CssVarsProvider } from '@mui/joy/styles';
-+ import Button from '@mui/joy/Button';
-
-function App() {
+export default function App() {
   return (
     <CssVarsProvider>
-+     <Button>Joy UI</Button>
+      <Button>Hello World</Button>
     </CssVarsProvider>
   );
 }
-
-export default App;
```

Comment:
I think that it would be great to have a working iframe codesandbox to start with like for Material UI. It can make playing with the API so much easier.

Reply:
I think the idea.

Reply:
Good call!